packages/cashier/src/components/resend-email-button-wrapper/__tests__/resend-email-button-wrapper.spec.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import ResendEmailButtonWrapper from '../resend-email-button-wrapper';
+
+jest.mock('Stores/connect.js', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+describe('<ResendEmailButtonWrapper/>', () => {
+    const resendVerificationEmail = jest.fn();
+
+    it('component should be rendered', () => {
+        const { container } = render(
```

Comment:
hey @vinu-deriv, can we avoid using container and querySelector?
RTL documentation has warnings in this regard (https://testing-library.com/docs/react-testing-library/api/#container-1), maybe we can use getByRole method?